mods/cnc/maps/nod06a/nod06a.lua

```diff
@@ -0,0 +1,227 @@
+Chn2ActorTriggerActivator = { Chn2Activator1, Chn2Activator2, Chn2Activator3 }
+Obj2ActorTriggerActivator = { Chn1Activator1, Chn1Activator2, Chn1Activator3, Chn1Activator4, Chn1Activator5, Chn2Activator1, Chn2Activator2, Chn2Activator3, Atk3Activator }
+
+Chn1Waypoints = { ChnEnrty.Location, waypoint5.Location }
+Chn2Waypoints = { ChnEnrty.Location, waypoint6.Location }
+Gdi3Waypoints = { waypoint1, waypoint3, waypoint7, waypoint8, waypoint9 }
+Gdi4Waypoints = { waypoint4, waypoint10, waypoint9, waypoint11, waypoint9, waypoint10 }
+Gdi5Waypoints = { waypoint1, waypoint4 }
+Gdi6Waypoints = { waypoint2, waypoints3 }
+
+Grd1TriggerFunctionTime = DateTime.Seconds(3)
+
+Grd1TriggerFunction = function()
+	MyActors = Utils.Take(2, GDI.GetActorsByType('mtnk'))
+	Utils.Do(MyActors, function(actor)
+		MovementAndHunt(actor, Gdi3Waypoints)
+	end)
+end
+
+Grd2TriggerFunction = function()
+	if not Grd2Switch then
+		for type, count in pairs({ ['e1'] = 2, ['e2'] = 1, ['jeep'] = 1 }) do
+			MyActors = Utils.Take(count, GDI.GetActorsByType(type))
+			Utils.Do(MyActors, function(actor)
+				MovementAndHunt(actor, Gdi4Waypoints)
+			end)
+		end
+		Grd2Swicth = true
+	end
+end
+
+Atk1TriggerFunction = function()
+	if not Atk1Switch then
+		for type, count in pairs({ ['e1'] = 3, ['e2'] = 3, ['jeep'] = 1 }) do
+			MyActors = Utils.Take(count, GDI.GetActorsByType(type))
+			Utils.Do(MyActors, function(actor)
+				MovementAndHunt(actor, Gdi5Waypoints)
+			end)
+		end
+		Atk1Switch = true
+	end
+end
+
+Atk2TriggerFunction = function()
+	if not Atk2Switch then
+		for type, count in pairs({ ['mtnk'] = 1, ['jeep'] = 1 }) do
+			MyActors = Utils.Take(count, GDI.GetActorsByType(type))
+			Utils.Do(MyActors, function(actor)
+				MovementAndHunt(actor, Gdi6Waypoints)
+			end)
+		end
+		Atk2Switch = true
+	end
+end
+
+Atk3TriggerFunction = function()
+	if not Atk3Switch then
+		Atk3Switch = true
+		if not Radar.IsDead then
+			local targets = Nod.GetGroundAttackers()
+			local target = targets[DateTime.GameTime % #targets + 1].CenterPosition
+
+			if target then
+				Radar.SendAirstrike(target, false, 256 - 28)
+			end
+		end
+	end
+end
+
+Chn1TriggerFunction = function()
+	local cargo = Reinforcements.ReinforceWithTransport(GDI, 'tran', Chn1Units, Chn1Waypoints, { waypoint14.Location })[2]
+	Utils.Do(cargo, function(actor)
+		IdleHunt(actor)
+	end)
+end
+
+Chn2TriggerFunction = function()
+	local cargo = Reinforcements.ReinforceWithTransport(GDI, 'tran', Chn2Units, Chn2Waypoints, { waypoint14.Location })[2]
+	Utils.Do(cargo, function(actor)
+		IdleHunt(actor)
+	end)
+end
+
+Obj2TriggerFunction = function()
+	Nod.MarkCompletedObjective(NodObjective2)
+	Reinforcements.Reinforce(Nod, Obj2Units, { Obj2UnitsEntry.Location, waypoint13.Location }, 15)
+end
+
+MovementAndHunt = function(unit, waypoints)
+	if unit ~= nil then
+		Utils.Do(waypoints, function(waypoint)
+			unit.AttackMove(waypoint.Location)
+		end)
+		IdleHunt(unit)
+	end
+end
+
+SendHarvester = function()
```

Comment:
The harvester is still idle (at least after a while). Try increasing the `SearchFromProcRadius` value on the Harvester in the map rules.